Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/LocationPage.tsx

```diff
@@ -61,9 +69,11 @@ export default function LocationPage(props: { LocationInfo: OpenXDA.Types.Locati
             error.push('Latitude needs to be between -180 and 180.')
         if (props.LocationInfo.Longitude != null && AssetAttributes.isRealNumber(props.LocationInfo.Longitude) && (props.LocationInfo.Longitude > 180 || props.LocationInfo.Longitude < -180))
             error.push('Longitude needs to be between -180 and 180.')
-
+        if (props.LocationInfo.ID == 0)
```

Comment:
I believe warnings show up with an yellow triangle. this really should not be a warning. Let's remove this for now.

Reply:
They asked for clarity on the time when this would save in item 3 of the email. If we avoid this, how do we provide this clarity?

Reply:
Given that all steps actually save as you click next This needs to be addressed at a higher level.
Either through addition of general comment to that effect or though documentation/training. But addressing it here (especially as a warning) won't be effective. We'll figure this item out later.

Reply:
Sounds good